src/FBGraphRequest.ts

```diff
@@ -56,24 +56,24 @@ class FBGraphRequest {
   /**
    * The optional config for the request.
    */
-  config: ?GraphRequestConfig;
+  config: GraphRequestConfig | undefined;
 
   /**
    * Called upon completion or failure of the request.
    */
-  callback: ?GraphRequestCallback;
+  callback: GraphRequestCallback | undefined;
```

Comment:
according to the constructor, `callback` will never be undefined,  so does `config`